examples/javascript/link-sdk.mjs

```diff
@@ -1,6 +1,11 @@
 import { CodatLink } from "https://link-sdk.codat.io";
 
 let companyId = "";
+const onConnection = (connection) =>
+  alert(`On connection callback  = ${connection.connectionId}`);
+const onClose = () => closeCallback();
```

Comment:
Good to declare closeCallback above this function, given it references it

Reply:
Can take a look after basketball this evening or tomorrow morning with Lucy if it can be done differently, but we have to control state of a bool inside the parent. 
that’s why it was passed as prop rather than declared within component. 

If not can leave a comment?